specification/cognitiveservices/OpenAI.Inference/models/completions/azure_chat_extensions.tsp

```diff
@@ -67,5 +88,86 @@ model AzureChatExtensionsMessageContext {
     course of generating a chat completions response that was augmented by capabilities from Azure OpenAI chat
     extensions.
   """)
-  messages?: ChatMessage[];
+  messages?: ChatResponseMessage[];
+}
+
+@added(ServiceApiVersions.v2023_12_01_Preview)
+@doc("A representation of the available options for the Azure OpenAI grounding enhancement.")
+model AzureChatGroundingEnhancementConfiguration {
+  @doc("Specifies whether the enhancement is enabled.")
+  enabled: boolean;
+}
+
+#suppress "@azure-tools/typespec-azure-core/casing-style" "established industry acronym"
+@added(ServiceApiVersions.v2023_12_01_Preview)
+@doc("A representation of the available options for the Azure OpenAI optical character recognition (OCR) enhancement.")
+model AzureChatOCREnhancementConfiguration {
+  @doc("Specifies whether the enhancement is enabled.")
```

Comment:
I think the documentation here suffers from the same as the above. Would be great to provide more details here, but it's a total nit pick too (in the current scheme of things 😄)

Reply:
Not a nit, it's important. Just also something I think we need to follow up on in parallel to getting (insufficiently documented and likely suboptimally structured) support available.